doc/guide/Makefile-guide.am

```diff
@@ -7,6 +7,7 @@ GUIDE_INCLUDES = \
 	doc/guide/api-base1.xml \
 	doc/guide/api-server.xml \
 	doc/guide/embedding.xml \
+	doc/guide/key-auth.xml \
```

Comment:
Could you rename this file to authentication.xml? That way we can expand it with other auth docs.